### ISSUE TITLE
`events.ts` assumes all messages sent between UI and main processes are arrays

### ISSUE THREAD

**Issue by fwextensions** [yuanqing/create-figma-plugin#180]
The code in `events.ts` assumes all messages sent within a plugin are arrays, and doesn't check for other types.  So if some code uses `postMessage()` directly, like this:

```js
window.parent.postMessage({ pluginMessage: { type: "msg", data: "this breaks create-figma-plugin" } }, "*")
```

Then this `events.ts` code will throw an error:

```js
figma.ui.onmessage = function ([name, ...args]
```

The library should check that the message has the right shape for it before trying to handle it, and ignore it if it doesn't.

**Comment by yuanqing** [comment]
Fixed in `2.6.1`

**Comment by fwextensions** [comment]
Thanks!  That was quick!

One question on this line: https://github.com/yuanqing/create-figma-plugin/commit/0b6933b435905ba89defe908096b9e8bc4bbd0a9#diff-7ca105005929c57993300fafef7cb34b0af6206a58472d777f9d60c397d3ad71R92

Why throw if the event isn't recognized?  I don't think there's any way to catch it, so any other code in the same plugin that sends an array with an unrecognized event name will cause a crash.
